include/hip/hip_runtime_api.h

```diff
@@ -6168,7 +6168,7 @@ hipError_t hipTexRefGetMipmapLevelClamp(
  *
  */
 DEPRECATED(DEPRECATED_MSG)
-hipError_t hipTexRefGetMipMappedArray(
+hipError_t hipTexRefGetMipmappedArray(
```

Comment:
There may need to be two functions going to one, this is a breaking change.

Reply:
to clarify you would like to have two functions same functionality with different names?